rosidl_typesupport_cpp/CMakeLists.txt

```diff
@@ -16,10 +16,12 @@ find_package(ament_cmake_ros REQUIRED)
 find_package(poco_vendor)
 find_package(Poco COMPONENTS Foundation)
 find_package(rcpputils REQUIRED)
-find_package(rosidl_generator_c REQUIRED)
+find_package(rosidl_runtime_c REQUIRED)
 
 link_directories(${Poco_LIBRARY_DIR})
 
+ament_export_dependencies(rosidl_runtime_c)
+ament_export_dependencies(rosidl_runtime_cpp)
```

Comment:
Why is this newly exported dependency necessary?

Reply:
This is still the same comment I had earlier:

> I simple change of name from generator to runtime is easy to understand. But when you add / remove dependencies please describe why you do it.

Why is this newly exported dependency necessary?